CMakeLists.txt

```diff
@@ -38,6 +66,7 @@ add_subdirectory(tests)
 # Documentation
 set(COMMON_PROJECT_DOMAIN ch.epfl.bluebrain)
 include(DoxygenRule)
+include(PackageConfig)
```

Comment:
Hmm, do we add PackageConfig to all the applications? It is not really a reusable library, I don't think users want to do a find_package(DisplayCluster)...